node/Dockerfile.s390x

```diff
@@ -12,23 +12,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG QEMU_IMAGE=calico/go-build:latest
-ARG BIRD_IMAGE=calico/bird:latest
+FROM alpine:3.11 as qemu
 
-FROM ${QEMU_IMAGE} as qemu
-FROM ${BIRD_IMAGE} as bird
+ARG QEMU_VERSION=6.1.0-8
+ARG QEMU_ARCHS="s390x"
 
-FROM calico/bpftool:v5.3-s390x as bpftool
+RUN apk --update add curl
 
-FROM s390x/alpine:3.8 as base
+# Enable non-native runs on amd64 architecture hosts
+RUN curl -L https://github.com/multiarch/qemu-user-static/releases/download/v${QEMU_VERSION}/qemu-${QEMU_ARCHS}-static.tar.gz | tar zxvf - -C /usr/bin &&\
+    chmod +x /usr/bin/qemu-s390x-static
 
-ARG ARCH=s390x
+FROM calico/bird:v0.3.3-188-g0196eee4-s390x as bird
```

Comment:
You should definitely use BIRD_IMAGE here, it gets propagated from https://github.com/projectcalico/calico/blob/ca43ac0dc253343bed5497b8d5a8876286a1b714/metadata.mk#L25-L26

Reply:
Yes, fixed it.